ide/static/js/content.js

```diff
@@ -228,6 +230,8 @@ class Content extends React.Component {
             downloadAnchor.download = response.name;
             downloadAnchor.href = response.url;
             downloadAnchor.click();
+            this.initialiseImportedNet(this.state.net,this.state.net_name);
```

Comment:
Main change i did in this file.
After exporting successfully,It initialise itself to the old state.

Reply:
All other changes are either prev commits(regarding VQA) or pep8 styling. :smile: